README.rst

```diff
@@ -224,13 +236,38 @@ Then build the documentation with::
 
 Then, direct your browser to ``_build/html/index.html``.
 
+
 Testing
 =======
 
 To test your Pyccel installation please run the script *tests/run_tests_py3.sh* (Unix), or *tests/run_tests.bat* (Windows).
 
 Continuous testing runs on Travis CI: <https://travis-ci.com/github/pyccel/pyccel>
 
+
+Pyccel Container Images
+=======================
+
+Pyccel container images are available through both Docker Hub (docker.io) and GitHub Container Registry (ghcr.io).
```

Comment:
the GitHub Container Registry

Reply:
This was copied from the other PR... Is "the" missing here?

Reply:
Yes, for completely correct English we should have a "the" here

Reply:
And why don't we need "the" before Docker Hub?

Reply:
Hmmm, maybe this is why I didn't signal it before. Registry needs "The" as there can be multiple different registries. There could also be more than one Docker Hub, but it's short enough to sound like a proper name. It might be right as it is but it sounds a little odd to me. Or we could put "the" before both

Reply:
OK, I've added "the" before "GitHub Container Registry" as you suggested.